doc/source/NewSiteConfigs.rst

```diff
@@ -361,54 +360,6 @@ The following instructions were used to prepare a basic Ubuntu 20.04 system as i
    apt install -y texlive
    apt install -y libcurl4-openssl-dev
    apt install -y libssl-dev
-   apt install -y mysql-server
-   apt install -y libmysqlclient-dev
-
-   # Python
-   apt install -y python3-dev python3-pip
-
-   # Exit root session
-   exit
-
-2. Log out and back in to be able to use the environment modules
-
-3. As regular user, set up the environment to build spack-stack environments
-
-This environment enables working with spack and building new software environments, as well as loading modules that are created by spack for building JEDI and UFS software.
-
-Prerequisites: Ubuntu 22.04 (one-off)
--------------------------------------
-
-The following instructions were used to prepare a basic Ubuntu 22.04 system as it is available on Amazon Web Services to build and install all of the environments available in spack-stack (see :numref:`Sections %s <Environments>`).
-
-1. Install basic OS packages as `root`
-
-.. code-block:: console
-
-   sudo su
-   apt-get update
-   apt-get upgrade
-
-   # Compilers (gcc@11.2.0)
-   apt install -y gcc g++ gfortran gdb
-
-   # lua/lmod module support
-   apt install -y lmod
-
-   # Do *not* install MPI with yum, this will be done with spack-stack
-
-   # Misc
-   apt install -y build-essential
-   apt install -y libkrb5-dev
-   apt install -y m4
-   apt install -y git
-   apt install -y git-lfs
-   apt install -y unzip
-   apt install -y automake
-   apt install -y xterm
-   apt install -y texlive
-   apt install -y libcurl4-openssl-dev
-   apt install -y libssl-dev
    apt install -y meson
```

Comment:
I don't know if it breaks anything later (I'm not sure how it would), but I do know that there is a meson package for 20.04 (verified with the `ubuntu:20.04` docker image), so this line works in all currently supported LTS releases.